package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "nx serve",
+    "start": "nx run-many --target=serve",
```

Comment:
We already have a `run-all` command here

Reply:
Is it intentional that the default script runs only front?